src/pages/signin/ResendValidationForm.js

```diff
@@ -87,7 +87,7 @@ class ResendValidationForm extends React.Component {
     }
 
     render() {
-        const isNewAccount = !this.props.account.accountExists;
+        const isNewAccount = !this.props.account.accountExists && !this.props.account.validateCodeExpired;
```

Comment:
I'm confused about this change or maybe how to interpret what a "new account" is in this context. 

`!accountExists` makes sense for a `isNewAccount` variable. But is there a situation where we would end up with `!accountExists && validatedCodeExpired`? 

Wouldn't `accountExists === true` if a validate code was issued in the first place? I could be mistaken but I thought any account with an `accountID` issued technically "exists" and is not "new".

Reply:
Well, when we set this validateCodeExpired, the existing `account` key in the localStorage doesn't `accountExists` key. It is undefined. 

Do you want me to set `accountExists` when I set `validateCodeExpired`? and if there's any specific response to know that the account doesn't exist.

Reply:
>when we set this validateCodeExpired, the existing account key in the localStorage doesn't accountExists key. It is undefined

Yes, this sounds incorrect to me. To reiterate, my point is that `validateCodeExpired` not existing doesn't seem like a requirement for a condition named `isNewAccount`. If you have a validate code at all then you do not have a "new account". So why is this check here?

>if there's any specific response to know that the account doesn't exist.

Sorry, didn't quite understand what you are asking - the response could return an `accountExists: true` but I'm not sure (we'd have to ask someone to verify what the backend is returning). Regardless, if we are setting `validateCodeExpired` to `true` then `accountExists` would also be `true`.